tests/tfm/tfm_psa_test/testcase.yaml

```diff
@@ -12,16 +12,29 @@ common:
       - "\\*\\*\\*\\*\\* PSA Architecture Test Suite - Version .* \\*\\*\\*\\*\\*"
       - "TOTAL FAILED *: 0"
 tests:
-  tfm.psa_test_protected_storage:
+  tfm.psa_test_protected_storage_lvl1:
```

Comment:
@torsteingrindvik Is this renaming here safe, or does it require CI update?

Reply:
Not safe. I have to update this on the test side.

Reply:
Let met know when you are ready with this @torsteingrindvik.

Reply:
> Let met know when you are ready with this @torsteingrindvik.

Yes, currently @Mangl10  is on it.